vsts/test-release-nuget.yaml

```diff
@@ -77,12 +77,7 @@ phases:
     displayName: build
     env:
       IOTHUB_CONN_STRING_CSHARP: $(IOTHUB-CONN-STRING-CSHARP)
-      IOTHUB_DEVICE_CONN_STRING: $(IOTHUB-DEVICE-CONN-STRING)
```

Comment:
Any reason we're removing these?

Reply:
We do not need the device connection string on the VSTS pipeline